client/src/href-builder.js

```diff
@@ -34,10 +34,10 @@ module.exports = function(config){
         component.parameters['__ocAcceptLanguage'] = lang;
       }
 
-      var versionSegment = !!component.version ? ('/' + component.version) : '',
-          registryUrl = clientRenderingEndpoint,
-          registrySegment = registryUrl.slice(-1) === '/' ? registryUrl : (registryUrl + '/'),
-          qs = !!component.parameters ? ('/?' + querystring.stringify(component.parameters)) : '';
+      const versionSegment = component.version ? ('/' + component.version) : '',
```

Comment:
I personally find those multiple assignments pretty annoying, can we have a pr in the future to remove them ? 😈 😅 😃